src/components/curriculum/CurriculumReactFundamentals.js

```diff
@@ -120,19 +120,21 @@ export const TargetAudienceList = () => (
 export const LearningObjectivesList = () => (
   <React.Fragment>
     <Li>
-      Understand the core principles and libraries to build React applications
-      using:{' '}
-      <code>
-        react, react-router, styled-components, storybook, redux, react-redux
-      </code>
+      Master React principles, such as the React composition model and the
+      one-way explicit data flow, to leverage React's full potential.
     </Li>
     <Li>
-      Create a solid foundation to leverage the React principles when tackling
-      complex React problems.
+      Understand how the most popular libraries to build React applications work
+      under the hood:{' '}
+      <code>react, react-dom, react-router, redux, react-redux</code>
     </Li>
-    <Li>Learn how to style React applications in an idiomatic way.</Li>
     <Li>
-      Understand different state management approaches in the React ecosystem.
+      Create a solid foundation so in the future you can learn quickly advanced
+      patterns and techniques as you progress in your career as React developer.
```

Comment:
Create a solid foundation so in future you can quickly learn advanced
patterns and techniques as you progress in your career as React developer.